IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcAlignmentCant/Documentation.md

```diff
@@ -1 +1,31 @@
 An _IfcAlignmentCant_ is a lateral inclination profile defined along the horizontal alignment. All points defined in this profile have two coordinate values. The first value is the distance along the horizontal alignment, and the second value is the height relative to the projection of the point along vertical alignment.
+
+
+**Cant as a railway specificity**
+
+!["Why cant is applied"](../../figures/ifcalignmentcant_wojtczak_2018.png "Figure 1 &mdash; Cant - compensation for lateral acceleration.")
+
+&gt;NOTE figure was first published in (Basics of) Railway Curve Kinematics Presentation, (Wojtczak 2018)
+
+The diagram in the figure above shows the principle of compensating part of the lateral acceleration by elevating the outer rail relative to the inner rail of a track.
+
+Almost always the following principles for the placement of horizontal segments and cant segments are implemented:
+* Straight horizontal line: no applied cant
+* Horizontal circular arc: constant applied cant
+* Horizontal transition bend: variation of applied cant according to a defined formula
+
+In specific difficult topographic environments like mountain lines the extension of a cant transition may differ from the horizontal transition. So far this has only been found for clothoids and is either a so called shortened ramp (can transition is shorter than the corresponding horizontal curvature transition) or a so called “scissor” ramp (overlapping variations of cant on both rails with opposite direction). 
+
+Application of high-performance horizontal transition bends is always combined with a cant transition with identical start and end location.
+
+In real world designs, such as slow speed sections in railway stations, exceptions to the above principles can be found. It might even happen that the inner rail is elevated relative to the outer rail.
+
+Detailed geometric modelling of the superelevation has a low priority in the existing processes in most rail companies. This will very probably change with the introduction of 3D modelling in rail design. The following list shows some use cases.
```

Comment:
```suggestion
Detailed geometric modelling of the superelevation has a low priority in the existing processes in most rail companies. This will very probably change with the introduction of 3D modelling in rail design. The following list shows some use cases:
```